neptune/new/internal/backends/neptune_backend_mock.py

```diff
@@ -510,8 +511,14 @@ def visit_artifact(self, _: Artifact) -> AttributeType:
         def visit_namespace(self, _: Namespace) -> AttributeType:
             raise NotImplementedError
 
+        def copy_value(
+            self, source_type: Type[Attribute], source_path: List[str]
+        ) -> AttributeType:
+            raise NotImplementedError
```

Comment:
We test it in non mocked backend, right?

Reply:
Sure we do, but that's `AttributeTypeConverterValueVisitor` used to build a "backend" attribute type; we never get attributes of type "Copy" from backend, and we never store it in Mock's map.